Shaders/deferred_light/deferred_light.frag.glsl

```diff
@@ -373,12 +396,6 @@ void main() {
 		#endif
 	#endif
 
-	#ifdef _VoxelAOvar
-	#ifdef _VoxelShadow
-	svisibility *= 1.0 - traceShadow(voxels, voxpos, sunDir);
```

Comment:
Why remove this? Is `_VoxelShadow` still working?

Reply:
I don't know I can't see a difference when activated or not.
I have set these functions to use this feature even with global illumination but that was just a bet.
No impairing anyway.